fingertip/util/filesystem.py

```diff
@@ -0,0 +1,99 @@
+# Licensed under GNU General Public License v3 or later, see COPYING.
```

Comment:
What's the reasoning to have this one separate from both `util.reflink` and `plugins/filesystem`?

Reply:
If it is fine to live in reflink, I am fine to add it there. But this sounded a bit unrelated to reflinking only. I wanted to avoid it having this in "plugins" as it is called directly from `main()`.

Reply:
Well, everything in this file sans `storage_schedule_cleanup` is only here because fingertip needs reflinking and most filesystems don't support it, esp. by default. I'm for moving it to `util/reflink`.

`storage_schedule_cleanup` can live in `plugins.cleanup`. if you're concerned with someone triggering it from the command line, don't be =) prefix it with an underscore if you really want to highlight that it's private.

Reply:
For the first part. I agree.
For the second one, I am not sure if we should just call from main something being it in "random" plugin. But if you find it fine, I will move it there too. For now, it is in the `utils/reflink`.